### PR TITLE
Gh 149 swagger get account balance rebased on 168

### PR DESCRIPTION
**Pull request by sennui**

### CODE REVIEW

File: apps/aehttp/src/aehttp_dispatch_ext.erl [hunk 1]
```diff
@@ -3,6 +3,8 @@
 -export([handle_request/3]).
 
 -compile({parse_transform, lager_transform}).
+-include("../../aecore/include/common.hrl").
```
Comment (sennui) [inline apps/aehttp/src/aehttp_dispatch_ext.erl]: this should work without jumping between dirs. Are deps set up correctly?

File: apps/aehttp/src/aehttp_dispatch_ext.erl [hunk 2]
```diff
@@ -90,6 +92,18 @@ handle_request('PostBlock', Req, _Context) ->
...
 
+handle_request('GetAccountBalance', Req, _Context) ->
+    Pubkey = maps:get('pub_key', Req),
+    {ok, LastBlock} = aec_chain:top(),
+    Trees = aec_blocks:trees(LastBlock),
+    AccountsTree = aec_trees:accounts(Trees),
+    case aec_accounts:get(Pubkey, AccountsTree) of
+        {ok, #account{balance = B}} ->
+            {200, [], #{balance => B}};
+        _ ->
```
Comment (lucafavatella) [inline apps/aehttp/src/aehttp_dispatch_ext.erl]: Open ended case. In future we can make more specific.

File: apps/aehttp/priv/swagger.json [hunk 3]
```diff
@@ -151,6 +151,37 @@
...
+        "tags" : [ "external" ],
+        "description" : "Get account's balance",
+        "operationId" : "GetAccountBalance",
+        "consumes" : [ "application/json" ],
+        "produces" : [ "application/json" ],
+        "parameters" : [ {
+          "name" : "pub_key",
+          "in" : "query",
+          "description" : "Public key to extract balance for",
+          "required" : true,
```
Comment (lucafavatella) [inline apps/aehttp/priv/swagger.json]: This means that client needs to know pubkey before enquiring. How does client know key?
Alternative would be to make this parameter optional, and if not specified simply call `aec_keys:pubkey()`

Comment (sennui) [reply apps/aehttp/priv/swagger.json]: @lucafavatella I am merging this, @velzevur please add /getbalance going to current account

**Pull request merged**